### ISSUE TITLE
Scaling issues of a non-public discussion approach

### ISSUE THREAD

**Issue by novoid** [ag91/emacs-buddy#3]
Hi,

I really do like the basic idea of helping GNN Emacs beginners with issues in form of a buddy-concept.

However, what makes me think is that this does not scale well because of the non-public nature of the proposed method as far as I understood it. In my personal experience, the vast majority of beginner questions are not unique questions to this very particular person starting with GNU Emacs. Therefore, the volunteers do invest their time with recurring beginners questions that could be answered once and published on the web so that other beginners are able to help themselves by using a search engine and others are able to link to those threads.

Therefore, I personally would prefer an approach like: "Ask any (non-specific general) question in one of the following places where other people might profit from the evolving message thread via Internet search. The volunteers listed at emacs-buddy can be @-mentioned via following user handles to help with particular questions."

What are your thoughts about that?

_Side-note: [I would love to see a revival of the Usenet for that purpose instead of using ephemeral web forums](https://karl-voit.at/2020/10/23/avoid-web-forums/) but this is a different discusssion._

**Comment by phikal** [comment]
Wouldn't that be something like the [help-gnu-emacs](https://lists.gnu.org/mailman/listinfo/help-gnu-emacs) mailing list?

**Comment by ag91** [comment]
TL;DR: I see the buddy as a companion that you can have a creative chat and that will try to help you to achieve Emacs enlightenment, not your personal problem solver. (And I am not concerned about scalability, because it is about creating a personal interaction through our commonality: enjoying Emacs.)

Thanks for your interest! Totally, this initiative is not about scaling. It is about making your learning of Emacs more personal.
An example of interaction I wish for:

> **EmacsUserLookingForBuddy**:  I am a physiotherapist, I struggle keeping track of my clients and they told me Org Mode is a great tool for that. I started with Emacs tutorial, but now how can I handle my clients??
> **EmacsBuddy**: cool! You can find a great Org Mode tutorial here and there is also a Reddit channel there. I can help you better if you tell me some more about what is your struggle with clients? For example, you struggle keeping track of communication or timetable or ....? By the way, I got a back pain: any chance you have a trick for that?!
> -- continues --

So my idea of a buddy should be to nudge new users towards Emacs enlightenment focusing on the why they started using Emacs. 
A forum or a mailing list is more about solving a particular problem you have, and you are totally spot on: that should be visible to everybody. (Indeed a good idea for a buddy to show how to move a conversation from private to a public channel to show the new user when is a good time to ask on a public forum!)
If I use Emacs to keep track of my weird hobby, I may find it helpful to get the point of view of my buddy on how to make the best out of my editor for that. And I can also just start chatting about something unrelated with the problem because I am enjoying the conversation.
Ideally you can make friends :D

**Comment by novoid** [comment]
Maybe you should think of adding a statement like that to the README?
(You may close this issue any time.)

**Comment by ag91** [comment]
You are right, thanks for making me write it down! I will do that and then close it :D
